editor/modes/visual-editor/style.scss

```diff
@@ -463,7 +464,7 @@
 		content: '';
 		position: absolute;
 		width: 100%;
-		height: 10px;
+		height: 44px;
```

Comment:
What was the reason for this change? Was it that it was too difficult to trigger the hover effect? The issue with this is that it aggravates the difficulty in selecting blocks which are short (like single-line paragraphs). The 44px was intended for hover effect only, intentionally overlapping some of the content of the block so that it would be slightly less easy to unhover the between-inserter ([related](http://bjk5.com/post/44698559168/breaking-down-amazons-mega-dropdown)). The natural height would be closer to `28px`. At the very least, if we keep this, we need to remove the now-redundant hover style override:

https://github.com/WordPress/gutenberg/blob/baa0ff79be45860c3352aba03e15f0b9296af20a/editor/components/block-list/style.scss#L415-L417

Reply:
I can't speak to the intent of this particular code, but wanted to point in the direction of #4312, in case it's relevant.

Reply:
See #4384